Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -59,11 +59,25 @@ final class DemoMerchantAPI {
         guard let url = buildBaseURL(with: "/orders/\(orderID)/capture", selectedMerchantIntegration: selectedMerchantIntegration) else {
             throw URLResponseError.invalidURL
         }
-        
+
         let urlRequest = buildURLRequest(method: "POST", url: url, body: EmptyBodyParams())
         let data = try await data(for: urlRequest)
         return try parse(from: data)
     }
+
+    func captureOrderWithPaymentToken<T: Encodable>(
```

Comment:
Because capture with vault without purchase flow requires parameters. I guess I can change call to this with empty body params for non vault without purchase flow

Reply:
Addressed in dda4eaefa72814221f4d270c1100109786fb3b0e